### DISCUSSION TITLE
I'm too stupid to get Disney+ cropped in Edge

### DISCUSSION THREAD

**Discussion by dashVid** [tamius-han/ultrawidify#157] [Q&A]
Hi. I'm really sorry, I tried. But I don't understand how to get Disney+ to enlarge the player/video in Edge.

I tried the 1-10 in n-th parent part, but that didn't work.

Could I ask for settings I can import to the extension to get Disney+ working, please?

Appreciate it. Thank you

**Comment by tamius-han** [comment]
> Could I ask for settings I can import to the extension to get Disney+ working, please?

Can you get me _your_ settings file first so I can compare things on my end?

> I tried the 1-10 in n-th parent part, but that didn't work.

Are both checkboxes checked?

Anyway, before I paste the settings, there's two caveats:

* Disney+ is not available in my country, so I couldn't test whether the settings work
* In some random cases that I cannot figure out, there's a chance that the browser will not attempt to load the extension at all? It's a strange thing, and as I said ... no disney+ in my country unless I fork over the money _both_ for D+ sub and a decent VPN, so with Disney+ I'm really going by what people tell me.

Settings:
[ultrawidify-settings.zip](https://github.com/tamius-han/ultrawidify/files/6760393/ultrawidify-settings.zip)

Please let me know if they work.
